src/main/java/pl/poznan/put/cs/scenariochecker/transformations/CountSpecialStepsScenarioStrategy.java

```diff
@@ -9,7 +9,12 @@
 
     @Override
     public void processScenario(Scenario scenario) {
-        scenario.setNumberOfSpecialSteps(countSteps(scenario.getSteps()));
+        scenario.setNumberOfSpecialSteps(changeValue(scenario.getSteps()));
+    }
+
+    private int changeValue(List<Step> steps){
+        int i = countSteps(steps);
+        return i-1;
```

Comment:
Instead of adding function that does only subtraction of 1, you could change `return (int) (1 + deepCount);` to `return (int) ScenarioHelper.isSpecialStep(this.step) + deepCount` or something like this, maybe you wil need change a little bit ScenarioHelper or wrap this content in a temporary step. Maybe you could start from write unit tests to see the best way - TDD ;) Look for "IF.. " subStep:{} you want get from function one instead of two :)